lib/compute/agent-nodes.ts

```diff
@@ -16,59 +16,55 @@ export class AgentNodes {
     // Refer: https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/ec2/model/InstanceType.html for instance types
     readonly AL2_X64: AgentNodeProps;
 
-    readonly AL2_X64_DOCKER_1: AgentNodeProps;
+    readonly AL2_X64_DOCKER_HOST: AgentNodeProps;
 
     readonly AL2_ARM64: AgentNodeProps;
 
-    readonly AL2_ARM64_DOCKER_1: AgentNodeProps;
+    readonly AL2_ARM64_DOCKER_HOST: AgentNodeProps;
 
-    readonly UBUNTU_X64_DOCKER: AgentNodeProps;
+    readonly UBUNTU_X64_DOCKER_BUILDER: AgentNodeProps;
 
     constructor(stack: Stack) {
       this.AL2_X64 = {
-        workerLabelString: 'AL2-X64',
+        workerLabelString: 'Jenkins-Agent-al2-x64-c54xlarge-Single-Host',
```

Comment:
Everything is jenkins here. How about we remove `Jenkins` form labels.

Reply:
Our jobs look for this label with `Jenkins` keyword in it. If we are removing here then let us make sure we update our job definitions as well.

Reply:
There are a lot of existing pipeline files using this format for Docker-Host, and I am more support of adding `Jenkins-Agent` because that is clear to tell you this is an agent node, not a container.